magma/interface.py

```diff
@@ -238,27 +214,24 @@ def __str__(cls):
         args = []
         for i, arg in enumerate(cls.Decl):
             if i % 2 == 0:
-                args.append('"{}"'.format(arg))
+                args.append(f"\"{arg}\"")
             else:
                 args.append(str(arg))
-        return ', '.join(args)
+        return ", ".join(args)
 
     def __eq__(cls, rhs):
-        if not isinstance(rhs, InterfaceKind): return False
-
-        if cls.Decl != rhs.Decl: return False
+        if not isinstance(rhs, InterfaceKind):
+            return False
+        if cls.Decl != rhs.Decl:
+            return False
         return True
```

Comment:
I think we could simplify to just `return cls.Decl == rhs.Decl`

Reply:
Done